fe/fe-core/src/main/java/org/apache/doris/load/routineload/RoutineLoadTaskScheduler.java

```diff
@@ -289,6 +289,13 @@ private void submitTask(long beId, TRoutineLoadTask tTask) throws LoadException
     private boolean allocateTaskToBe(RoutineLoadTaskInfo routineLoadTaskInfo) throws LoadException {
         long beId = routineLoadManager.getAvailableBeForTask(routineLoadTaskInfo.getPreviousBeId(), routineLoadTaskInfo.getClusterName());
         if (beId == -1L) {
```

Comment:
I think we can just modify the `getAvailableBeForTask()` method?
In that method: check if previous Backend is alive, if yes, go on. If no, find a new one.

Reply:
So that we don't need to add config for this.

Reply:
Done.